open-api/rest-catalog-open-api.yaml

```diff
@@ -222,7 +222,10 @@ paths:
           description:
             An optional namespace, underneath which to list namespaces.
             If not provided or empty, all top-level namespaces should be listed.
-            If parent is a multipart namespace, the parts must be separated by the unit separator (`0x1F`) byte.
+            If parent is a multipart namespace, the parts should be separated by the namespace separator as
+            indicated via the /config override `rest-namespace-separator`, which defaults to the unit separator (`0x1F`) byte.
+            To be compatible with older clients, servers have to use `0x1F` as a fallback even when advertising a different
+            namespace separator to clients.
```

Comment:
Thanks for the update @nastra . Sorry to be nit-picky, but I believe this text is still not clear enough. I'd suggest `To be compatible with older clients, servers should use both the advertised separator and`0x1F` as a valid srparators when parsing namespaces.` (feel free to edit)

Reply:
My point is that it is not clean what is "fallback" in this case. I propose to treat both old and new separators equally (always respect both, even when intermixed)

Reply:
updated

Reply:
thx - LGTM :+1:

Reply:
How does the server know what (single) separator is effective for a particular request?